charts/kvisor/templates/secret.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if not .Values.castai.apiKeySecretRef }}
+{{- if .Values.castai.apiKey }}
```

Comment:
IMHO we should stick to the `not .Values.castai.apiKeySecretRef` condition, as otherwise it is quite easy to misconfigure the helm chart and not get an error.

Reply:
I wanted to make castai api key optional to allow users test with local clickhouse.